test/verify/check-networkmanager-wireguard

```diff
@@ -84,24 +84,34 @@ class TestWireGuard(packagelib.PackageCase, netlib.NetworkCase):
         b.set_input_text("#network-wireguard-settings-listen-port-input", str(m1_port))
 
         # check ip addresses
+        # ubuntu-2204 and debian-stable shows wrong warning
```

Comment:
one specific case for ubuntu-2204 is that NM gives different(wrong) error message when an invalid IP address is sent, the error should say invalid ipv4
<img width="837" height="328" alt="image" src="https://github.com/user-attachments/assets/c2821a75-f90b-4417-b1a1-94b9269ebc5f" />

Reply:
Hmm, shouldn't we catch invalid address syntax before calling NM?